packages/fuel-indexer-macros/src/indexer.rs

```diff
@@ -105,6 +105,15 @@ fn process_fn_items(
                 }
             }
         })
+        // Add an arm for a unit struct in case of empty data fields
```

Comment:
@deekerno I think we can remove this comment (I think the code makes it clear what's goin on)